bot/resources/evergreen/trivia_quiz.json

```
@@ -0,0 +1,221 @@
+      "id": 116,
+      "question": "The Vally Of The Kings is located in which country?",
+      "answer": "Egypt",
+      "info": "The Valley of the Kings, also known as the Valley of the Gates of the Kings, is a valley in Egypt where, for a period of nearly 500 years from the 16th to 11th century BC, rock cut tombs were excavated for the pharaohs and powerful nobles of the New Kingdom (the Eighteenth to the Twentieth Dynasties of Ancient Egypt)."
+    },
+    {
+      "id": 117,
+      "question": "Diamonds are always nice in Minecraft, but can you name the \"Diamond Capital in the World\"?",
+      "answer": "Antwerp",
+      "info": "Antwerp, Belgium is where 60-80% of the world's diamonds are cut and traded, and is known as the \"Diamond Capital of the World.\""
+    },
+    {
+      "id": 118,
+      "question": "Where is the \"International Court Of Justice\" located at?",
+      "answer": "Hague",
+      "info": ""
+    },
+    {
+      "id": 119,
+      "question": "In which country is Bali located in?",
+      "answer": "Indonesia",
+      "info": ""
+    },
+    {
+      "id": 120,
+      "question": "What country is the world's largest coral reef system, the \"Great Barrier Reef\", located in?",
+      "answer": "Australia",
+      "info": "The Great Barrier Reef is the world's largest coral reef system composed of over 2,900 individual reefs and 900 islands stretching for over 2,300 kilometres (1,400 mi) over an area of approximately 344,400 square kilometres (133,000 sq mi). The reef is located in the Coral Sea, off the coast of Queensland, Australia."
+    },
+    {
+      "id": 121,
+      "question": "When did the First World War start?",
+      "answer": "1914",
+      "info": "The first world war began in August 1914. It was directly triggered by the assassination of the Austrian archduke, Franz Ferdinand and his wife, on 28th June 1914 by Bosnian revolutionary, Gavrilo Princip. This event was, however, simply the trigger that set off declarations of war."
+    },
+    {
+      "id": 122,
+      "question": "Which is the largest desert in the world (exclude cold deserts.) ?",
+      "answer": "Sahara",
+      "info": "The Sahara Desert covers 3.6 million square miles. It is almost the same size as the United States or China. There are sand dunes in the Sahara as tall as 590 feet."
+    },
+    {
+      "id": 123,
+      "question": "Who lived at 221B, Baker Street, London?",
+      "answer": "Sherlock Holmes",
+      "info": ""
+    },
+    {
+      "id": 124,
+      "question": "When did the Second World War end?",
+      "answer": "1945",
+      "info": "World War 2 ended with the unconditional surrender of the Axis powers. On 8 May 1945, the Allies accepted Germany's surrender, about a week after Adolf Hitler had committed suicide. VE Day – Victory in Europe celebrates the end of the Second World War on 8 May 1945."
+    },
+    {
+      "id": 125,
+      "question": "What is the name of the largest Dam in the world?",
```

```suggestion
      "question": "What is the name of the largest dam in the world?",
```
Missed this one on the first review, whoops.